cinema/tests/test_movie_api.py

```diff
@@ -157,3 +156,191 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieAPITests(TestCase):
```

Comment:
The `UnauthenticatedMovieAPITests` class tests the behavior of the API when accessed without authentication. This is important for ensuring that the API enforces authentication requirements.